### PR TITLE
Docs Update: Updating AppKit overview doc content

### PR DESCRIPTION
**Pull request by rohit-710**
## Description

The overview page for AppKit was quite minimal in terms of both the description of AppKit and the overview of its features. This PR addresses that. 

## Tests

Tested locally with Docusaurus.

### CODE REVIEW

File: docs/appkit/overview.mdx [hunk 1]
```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
...
 ![AppKit banner](../../static/img/appkit.png)
 
-Some of the key features of the AppKit SDK include:
+#### Framework Agnostic
+Integrate effortlessly with popular frameworks like React, Vue, Svelte, vanilla JavaScript, and more.
+
+#### EIP-6963 Support
+Seamlessly connect browser extension wallets for an enhanced user experience. Learn more about **EIP-6963 [here](https://eips.ethereum.org/EIPS/eip-6963).**
+
+#### Email Wallets
```
Comment (rtomas) [inline docs/appkit/overview.mdx:121]: you are repeating the some emails here

#### Email & Social Login
Expand your app’s reach by enabling users to connect using their email or social accounts. Implement these features with minimal code and leverage Smart Account functionality for advanced capabilities.

File: docs/appkit/overview.mdx [hunk 2]
```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
...
+#### Customizable UI
+Choose between light and dark modes, accent colors, or fully customize the interface with your own branding.
+
+#### Multi-Chain Ready
+Easily configure support for EVM and non-EVM chains, giving users access to multiple blockchain networks.
+
+#### Hundreds of Wallets
+Connect with popular wallets like MetaMask, Rainbow, Coinbase, and many more to maximize accessibility.
+
+#### Transaction History
```
Comment (rtomas) [inline docs/appkit/overview.mdx:136]: you have two times 
#### Transaction History

File: docs/appkit/overview.mdx [hunk 3]
```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
...
 
-- **Support for multiple frameworks.** Easily integrate with React, Vue, Svelte, vanilla JavaScript, and more.
-- **[EIP-6963.](https://eips.ethereum.org/EIPS/eip-6963) support** for browser extension wallets.
-- **Email Wallets.** Allow users to seamlessly connect to dapps with their email address.
-- **On-Ramp.** Enable users to buy crypto with fiat currency.
-- **Customizable UI**. Choose between light and dark modes and accent colors, or even infuse it with your own branding.
-- **Multi-chain ready.** Easily set up support for any EVM chain.
-- **Hundreds of Wallets.** Connect with MetaMask, Rainbow, Coinbase and many more.
-- **Transaction History.** View all transactions from the account view.
+#### Telegram MiniApps
```
Comment (rtomas) [inline docs/appkit/overview.mdx:160]: we are not supporting social logins with tg mini apps ... or we are ?

**Review by rtomas — APPROVED** [review]
LGTM

File: docs/appkit/overview.mdx (hunk 3 above)
Comment (rohit-710) [reply docs/appkit/overview.mdx:160]: We support social logins

File: docs/appkit/overview.mdx (hunk 2 above)
Comment (rohit-710) [reply docs/appkit/overview.mdx:136]: Thanks for catching this. Fixed

**Pull request merged**